controller/user-controller.js

```diff
@@ -0,0 +1,41 @@
+const {userService} = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname, skintype} =req.body;
+
+    await userService.signUp(email, password, firstname, lastname, skintype);
+
+    if (!email|| !password || !firstname || !lastname){
+      throw new Error("KEY Error!");
+    }
+
+    return res.status(201).send({message: "SIGNUP_SUCCESS"});
+  }catch (err) {
+    res.status(err.statusCode || 400).json({message: err.message});
+  }
+};
+
+const signIn = async (req, res) => {
+  try{
+    const {email, password }= req.body;
+
+   const accessToken = await userService.signIn(email,password);
+   
+   
+   if( !eamil || !password ){
+    throw new Error("SIGN IN FAILED!");
+   }
+
+   return res.status(200).json({accessToken: accessToken});
+  } catch (err){
+    res.status(er.statusCode || 400).json ({message: err.message});
```

Comment:
해당 부분에도 오탈자가 있네요~!
프로그래밍의 치명적인 오류는 쉽게 찾을 수 없는 오타 (Human error)에서 비롯될 수 있으니, 반드시 내가 작성한 코드가 올바르게 동작하는지 다양한 툴을 이용하여 테스트하는 것도 중요합니다!

```suggestion
    res.status(err.statusCode || 400).json({message: err.message});
```